connect/runtime/src/test/java/org/apache/kafka/connect/runtime/WorkerSourceTaskTest.java

```diff
@@ -439,6 +439,26 @@ public void testSendRecordsPropagatesTimestamp() throws Exception {
     }
 
     @Test
+    public void testSendRecordsNoTimestamp() throws Exception {
```

Comment:
Can you also add a test case for the invalid case? You can use `@Test(expected = CorruptRecordException.class)`